### DISCUSSION TITLE
[Question] ASF timeline

### DISCUSSION THREAD

**Discussion by adipolak** [apache/fluss#58] [General]
In the latest FlinkForward, Jark Wu discussed contributing flus to ASF. Is there a deadline/time for that?

**Comment by wuchong** [comment]
Hi Adi, thanks for your attention. Fluss plans to donate to the ASF after iterating through one or two releases and having some initial contributors and diversity.

**Comment by XuQianJin-Stars** [comment]
> In the latest FlinkForward, Jark Wu discussed contributing flus to ASF. Is there a deadline/time for that?

`flus` -> `fluss` ?
